src/components/Footer.js

```diff
@@ -0,0 +1,142 @@
+import React from "react";
+import './footer.css'
+import MobileIcon from "./images/pngwing.com (5).png"
+import VkIcon from "./images/pngwing.com (1).png"
+import FbIcon from "./images/pngwing.com (2).png"
+import OkIcon from "./images/pngwing.com (3).png"
+import IgIcon from "./images/pngwing.com (4).png"
+import Arrow from "./images/arrow.png"
+
+const Footer = () => {
+
+    const elements = {
+        "sales": {
+            title: ["Продажа"],
```

Comment:
value of field title can be just string, not array. also u should change logic in reurnable elements